yarn.lock

```diff
@@ -2393,7 +2393,11 @@ cross-spawn@5.1.0, cross-spawn@^5.0.1, cross-spawn@^5.1.0:
     shebang-command "^1.2.0"
     which "^1.2.9"
 
+<<<<<<< HEAD
```

Comment:
Looks like this file has conflicts that need to be fixed.  An easy way it to delete the yarn.lock file and then regenerate it